frontend/app/views/shared/_header_repository.html.erb

```diff
@@ -194,7 +194,7 @@
                 </div>
               </li>
               <% if ArchivesSpaceHelp.enabled? %>
-                <li><%= link_to_help :link_opts => {"data-placement" => (session[:user] ? "left" : "bottom")} %></li>
```

Comment:
I wonder what the reasoning for this was?

Reply:
Yeah that is odd. I didn’t look into the history. But the staff app past login isn’t available to non-users, so it seems to be to have always been set to the left.